src/gossip/data.zig

```diff
@@ -1248,11 +1247,11 @@ pub const ContactInfo = struct {
             .wallclock = wallclock,
             .outset = outset,
             .shred_version = shred_version,
-            .version = ClientVersion.default(),
+            .version = ClientVersion.CURRENT,
             .addrs = ArrayList(IpAddr).init(allocator),
             .sockets = ArrayList(SocketEntry).init(allocator),
             .extensions = ArrayList(void).init(allocator),
-            .cache = socket_addrs_unspecified(),
+            .cache = socket_addrs_unspecified,
```

Comment:
This is already a default field, no need to initialize it twice. I'd rather we not have `socket_addrs_unspecified` and just,
```zig
.{SocketAddr.UNSPECIFIED} ** SOCKET_CACHE_SIZE,
```
for the default field instead?

Reply:
I wonder why it even is a default value if we never actually initialize the struct directly - how about I just remove the default value and `socket_addrs_unspecified`?

Reply:
You need it for the bincode skip I believe.

Reply:
ff9e83da8d8dccc577f257d83c1723405f3a1f30